xdp-filter/xdp-filter.c

```diff
@@ -592,9 +603,9 @@ int do_port(const void *cfg, const char *pin_root_path)
 		goto out;
 
 	if (opt->remove)
-		flags &= ~(opt->mode | opt->proto);
+		flags &= ~(mode | proto);
```

Comment:
Might as well just move these assignments up into the previous if() blocks instead of having two checks for opt->remove.

Reply:
I wanted to separate the default values and the flag change. Also, this would change the debug output order a little. But I merged them in the next version.